runtime/tests/port/si.c

```diff
@@ -2183,9 +2183,17 @@ j9sysinfo_test_get_l1dcache_line_size(struct J9PortLibrary *portLibrary)
 	cQuery.level = 1;
 	cQuery.cacheType = J9PORT_CACHEINFO_DCACHE;
 	rc = j9sysinfo_get_cache_info(&cQuery);
-#if !(defined(J9ARM) || defined(RISCV64))
+#if !(defined(J9ARM))
```

Comment:
Please remove the unnecessary pareentheses.

Reply:
Fixed.